### DISCUSSION TITLE
DNS name resolution

### DISCUSSION THREAD

**Discussion by tstevenson** [akvorado/akvorado#1166] [Q&A]
Any plan to add an option to resolve IPs to DNS names? Would be super handy. Might also be nice to have the option to resolve well-known L4 ports to protocol names.
Thanks.

**Comment by vincentbernat** [comment]
L4 ports to protocol names is "easy". The DNS, not so.

**Comment by tstevenson** [comment]
Thanks for the reply. For the L4 ports, just a static config block would work (eg, pre-populated and user can add additional ones). 

Understand that DNS is hard, but obviously highly useful - mapping numeric IPs (esp IPv6 addresses) to services/owners is work best left to the computers ;)

**Comment by bhicks69** [comment]
I'm also seeing where DNS resolution (especially in v6). I wonder if a solution using a host file would be an interim solution.
